wavelength-plugins/tasks.js

```diff
@@ -0,0 +1,91 @@
+/********************************
+ * Tasks (To-Do/Jobs) Plug-in	*
+ * Created for Pokemon Showdown *
+ * Created by Insist			*
+ ********************************/
+
+"use strict";
+
+function isDev(user) {
+	if (!user) return;
+	if (typeof user === "object") user = user.userid;
+	let dev = Db.devs.get(toId(user));
+	if (dev === 1) return true;
+	return false;
+}
+
+function alertDevs(message) {
+	let developers = Db.devs.keys();
+	for (let u in developers) {
+		if (!Users(developers[u]) || !Users(developers[u]).connected) continue;
+		Users(developers[u]).send(`|pm|~Developer Alert|~|/raw ${message}`);
+	}
+	if (Rooms(`development`)) Rooms(`development`).add(`|c|~Developer Alert|/raw ${message}`).update();
+}
+
+exports.commands = {
+	jobs: "tasks",
+	job: "tasks",
+	todo: "tasks",
+	task: "tasks",
+	tasks: {
+		new: "add",
+		issue: "add",
+		add: function (target, room, user) {
+			if (!isDev(user.userid) && !this.can("bypassall")) return false;
+			let [issue, ...description] = target.split(",").map(p => p.trim());
+			let task = Db.tasks.get("development", {issues: {}});
+			if (!issue || !description) return this.parse("/taskshelp");
+			if (task.issues[toId(issue)]) return this.errorReply(`This issue title already exists.`);
+			if (issue.length < 1 || issue.length > 30) return this.errorReply(`The issue title should not exceed 30 characters long. Feel free to continue in the description.`);
+			if (description.length < 1 || description.length > 100) return this.errorReply(`The description should not exceed 100 characters long.`);
+			task.issues[toId(issue)] = {"id": toId(issue), "issue": issue, "description": description, "employer": user.userid};
+			Db.tasks.set("development", task);
+			alertDevs(`${WL.nameColor(user.name, true, true)} has filed an issue. Issue: ${issue}. Description: ${description}.`);
+			return this.sendReply(`The task "${issue}" has been added to the server task list.`);
+		},
+
+		remove: "delete",
+		clear: "delete",
+		fixed: "delete",
+		delete: function (target, room, user) {
+			if (!isDev(user.userid) && !this.can("bypassall")) return false;
+			target = toId(target);
+			let task = Db.tasks.get("development", {issues: {}});
+			if (!target) return this.parse(`/taskshelp`);
+			if (!task.issues[target]) return this.errorReply(`The issue "${target}" has not been reported.`);
+			delete task.issues[target];
+			Db.tasks.set(task);
+			return this.sendReply(`The task "${target}" has been deleted.`);
+		},
+
+		"": "list",
+		tasks: "list",
+		task: "list",
+		list: function (fullCmd, room, user) {
+			if (!isDev(user.userid) && !this.can("bypassall")) return false;
+			if (!this.runBroadcast()) return;
+			if (this.broadcasting && room.id !== "development") return this.errorReply(`You may only broadcast this command in Development.`);
```

Comment:
combine these into one line ``if (room && room.id === 'development' && !this.runBroadcast()) return;`` that should cover it and prevent broadcast outside of dev + still return the box